docs/capi.rst

```diff
@@ -884,11 +884,37 @@ Functions
 
   Define a string external variable.
 
+.. c:function:: int yr_scanner_scan_mem_blocks(YR_SCANNER* scanner, YR_MEMORY_BLOCK_ITERATOR* iterator)
+
+  .. versionadded:: 3.8.0
+
+  Scan a memory buffer divided into one or more overlapping memory blocks
```

Comment:
The description suggest that the blocks are always overlapping, but that's not the case. Actually that's up to the implementor of the iterator. This is more accurate:

`Scan a memory buffer divided into one or more memory blocks (possibly overlapping)` 


There's also a typo in "accesssed".

Reply:
Okay. Changes. And thanks for spotting the typo!